fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexScrubberTest.java

```diff
@@ -469,19 +469,24 @@ void testScrubUnnestedIndex(boolean legacy) {
                         .useLegacyScrubber(legacy)
                         .build())
                 .build()) {
-            indexScrubber.scrubDanglingIndexEntries();
-            indexScrubber.scrubMissingIndexEntries();
+            if (!legacy) {
```

Comment:
Nit: 
Since most of the other tests are structured as a positive question. I.E.: 
```
if (legacy) {
} else {
}
```
Can we change this to positive to make the code slightly more predictable?